docker-app/qfieldcloud/core/tests/test_localized_projects.py

```diff
@@ -131,20 +138,11 @@ def test_localized_layers_and_missing_localized_layers(self):
         self.assertIsNotNone(processprojectfile_job.feedback)
 
         # Localized layers found in the localized datasets project
-        if self.project1.uses_legacy_storage:
-            available_shared_files = utils.get_project_files(
-                self.shared_datasets_project.id
-            )
-            available_localized_filenames = [
-                file.name for file in available_shared_files
-            ]
-
-        else:
-            available_localized_filenames = set(
-                File.objects.filter(
-                    project_id=self.shared_datasets_project.id
-                ).values_list("name", flat=True)
+        available_localized_filenames = set(
+            File.objects.filter(project_id=self.shared_datasets_project.id).values_list(
+                "name", flat=True
             )
+        )
```

Comment:
```suggestion
        available_localized_filenames = set(
            File.objects.filter(
                project_id=self.shared_datasets_project.id,
            ).values_list("name", flat=True)
        )
```

In those cases we try to make the filter line by line.